benchmark/main.cpp

```diff
@@ -184,7 +184,11 @@ do {                                                    \
     t.stop();                                           \
     ms = t.read_ms();                                   \
                                                         \
-    if (ret != 0) {                                     \
+    if (ret == MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED)\
+    {                                                   \
```

Comment:
I'm sorry, but I don't understand. What brace? What condition?

Reply:
As far as I know, in Mbed OS (or at least in our example applications) the braces should be on the same lines as the if condition. So it should be:

    if (condition) {

rather than:

    if (condition)
    {

My mind may have been clouded when I wrote the comment, because I agree it isn't clear. Most probably I wanted to write "... on the same line as the condition _above_".